### DISCUSSION TITLE
Should accesslogging.tf be moved to an account level module?

### DISCUSSION THREAD

**Discussion by ntwkninja** [defenseunicorns/delivery-aws-iac#223] [Q&A]
In this repository, examples typically inform how a set of modules can be used coherently. It seems the [access-logging.tf](https://github.com/defenseunicorns/delivery-aws-iac/blob/main/examples/complete/access-logging.tf) may be an anti-pattern because it provisions resources in our examples vs. moving resources into a module for reusability.

Proposal: Move access-logging.tf capabilities to it's own module; possibly for account-level / init of a new cloud environment but maybe not.

**Comment by RothAndrew** [comment]
There's an issue for creating a KMS module already: #175. I agree we should consider making one for a logging bucket as well.
